.github/workflows/changelog.yml

```diff
@@ -5,6 +5,8 @@ on:
     types: [opened, synchronize, labeled, unlabeled]
     branches:
       - main
+    paths:
+      - 'src/snowflake/snowpark/**'
```

Comment:
should we ignore the future modin path?

Reply:
Yes we should, but the modin directory doesn't exist yet. Should I add a TODO here linking to the epic for the JIRA?

Reply:
Sounds good to me with a TODO